### ISSUE TITLE
Bids-validator/cli not found

### ISSUE THREAD

**Issue by max-kat** [pBFSLab/DeepPrep#180]
Hi there! 

I just found your amazing upgrade of fMRIprep and want to use it on my new dataset!
I followed the installation steps and tried to analyse one of my participants, however deeprep always errors out and I cannot find the reason why. I even downloaded your toy example and also there it errors out. It seems to be a problem with BIDS... 

**Did I check BIDS standards?**
Yes with the online BIDS validator and my dataset is conform 

**What is my command?**
     
`docker run -ti --rm \
                  -v /home/kathoferm47/mnt/cogsci/userdata/juliacrone/2021_CPketamine/data/pre:/input \
                  -v /home/kathoferm47/mnt/cogsci/userdata/juliacrone/2021_CPketamine/data/pre/derivatives:/output \
                  -v /home/kathoferm47/mnt/cogsci/userdata/juliacrone/2021_CPketamine/data/fs_license.txt:/fs_license.txt \
                  pbfslab/deepprep:24.1.1 \
                  /input /output \
                  participant \
                  --bold_task_type resting \
                  --participant_label 'sub-011' \
                   --device cpu --cpus 20 \
                   --fs_license_file /opt/freesurfer/license.txt  
`

**What is the error?**

```
ERROR ~ Error executing process > 'deepprep_init'

Caused by:
  Process `deepprep_init` terminated with an error exit status (1)

Command executed:

  df -h
  
  gpu_schedule_lock.py executor
  
  input_bids_validator.py     --bids_dir /input     --exec_env docker     --participant_label sub-011     --skip_bids_validation False
  
  deepprep_init.py     --freesurfer_home /opt/freesurfer     --bids_dir /input     --output_dir /output     --subjects_dir /output/Recon     --bold_spaces fsaverage6     --bold_only False

Command exit status:
  1

Command output:
  Filesystem                   Size  Used Avail Use% Mounted on
  overlay                      439G  398G   19G  96% /
  tmpfs                         64M     0   64M   0% /dev
  tmpfs                        126G     0  126G   0% /sys/fs/cgroup
  shm                           64M     0   64M   0% /dev/shm
  //share.univie.ac.at/cogsci   24T   14T  9.8T  58% /input
  /dev/md0                     439G  398G   19G  96% /etc/hosts
  tmpfs                        126G     0  126G   0% /proc/acpi
  tmpfs                        126G     0  126G   0% /proc/scsi
  tmpfs                        126G     0  126G   0% /sys/firmware
  tmpfs                        126G     0  126G   0% /sys/devices/virtual/powercap
  set gpu lock
  connect to StrictRedis DB
  
   ERROR: This Error is from bids_validator.You can add --skip_bids_validation in command to skip it. 

Command error:
  tmpfs                        126G     0  126G   0% /sys/fs/cgroup
  shm                           64M     0   64M   0% /dev/shm
  //share.univie.ac.at/cogsci   24T   14T  9.8T  58% /input
  /dev/md0                     439G  398G   19G  96% /etc/hosts
  tmpfs                        126G     0  126G   0% /proc/acpi
  tmpfs                        126G     0  126G   0% /proc/scsi
  tmpfs                        126G     0  126G   0% /sys/firmware
  tmpfs                        126G     0  126G   0% /sys/devices/virtual/powercap
  set gpu lock
  connect to StrictRedis DB
  node:internal/modules/cjs/loader:1148
    throw err;
    ^
  
  Error: Cannot find module 'bids-validator/cli'
  Require stack:
  - /opt/conda/envs/deepprep/bin/bids-validator
      at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
      at Module._load (node:internal/modules/cjs/loader:986:27)
      at Module.require (node:internal/modules/cjs/loader:1233:19)
      at require (node:internal/modules/helpers:179:18)
      at Object.<anonymous> (/opt/conda/envs/deepprep/bin/bids-validator:18:30)
      at Module._compile (node:internal/modules/cjs/loader:1358:14)
      at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
      at Module.load (node:internal/modules/cjs/loader:1208:32)
      at Module._load (node:internal/modules/cjs/loader:1024:12)
      at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12) {
    code: 'MODULE_NOT_FOUND',
    requireStack: [ '/opt/conda/envs/deepprep/bin/bids-validator' ]
  }
  
  Node.js v20.16.0
  Traceback (most recent call last):
  
   ERROR: This Error is from bids_validator.You can add --skip_bids_validation in command to skip it. 
  
    File "/opt/DeepPrep/deepprep/nextflow/bin/input_bids_validator.py", line 27, in <module>
      validate_input_dir(args.exec_env, bids_dir, participant_label)
    File "/opt/conda/envs/deepprep/lib/python3.10/site-packages/fmriprep/utils/bids.py", line 237, in validate_input_dir
      subprocess.check_call(['bids-validator', str(bids_dir), '-c', temp.name])
    File "/opt/conda/envs/deepprep/lib/python3.10/subprocess.py", line 369, in check_call
      raise CalledProcessError(retcode, cmd)
  subprocess.CalledProcessError: Command '['bids-validator', '/input', '-c', '/tmp/tmpe6kubqty.json']' returned non-zero exit status 1.
  
  During handling of the above exception, another exception occurred:
  
  Traceback (most recent call last):
    File "/opt/DeepPrep/deepprep/nextflow/bin/input_bids_validator.py", line 33, in <module>
      raise (ChildProcessError, "ERROR: This Error is from bids_validator.You can add --skip_bids_validation in command to skip it.")
  TypeError: exceptions must derive from BaseException

Work dir:
  /output/WorkDir/nextflow/c6/fdf28fe80be0750119613794671d6c

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`

 -- Check '.nextflow.log' file for details
```

So basically deepprep creates the workflows but then just aborts. I am happy with any help cause I am also quite new to docker (did I do something wrong while calling docker?) 

Thanks for any help! 

Max
